tests/lib/external/snapd-testing-tools/tools/tests.pkgs.pacman.sh

```diff
@@ -25,7 +25,7 @@ remap_one() {
             echo "python-gobject"
             ;;
         test-snapd-pkg-1)
-            echo "freeglut"
+            echo "cmus"
```

Comment:
Same

Reply:
everything under tests/lib/external/snapd-testing-tools is the same